### ISSUE TITLE
Wrong indentation

### ISSUE THREAD

**Issue by danieletorelli** [scalameta/scalafmt#1439]
- **Version**: 2.0.0-RC8
- **Integration**: IntelliJ + SBT plugin
- **Configuration**:
```
version = 2.0.0-RC8
style = intellij
maxColumn = 120
lineEndings = unix
align = most
align.openParenCallSite = false
align.openParenDefnSite = false
align.tokens = [
  {code = "<-"},
  {code = "=>", owner = "Case"},
  {code = "%", owner = "Term.ApplyInfix"},
  {code = "%%", owner = "Term.ApplyInfix"}
]
continuationIndent.defnSite = 2
rewrite.rules = [RedundantBraces, RedundantParens, SortImports, SortModifiers, PreferCurlyFors]
rewrite.sortModifiers.order = ["override", "private", "protected", "implicit", "final", "sealed", "abstract", "lazy"]
newlines.sometimesBeforeColonInMethodReturnType = false
unindentTopLevelOperators = true
spaces.inImportCurlyBraces = true
danglingParentheses = true
includeCurlyBraceInSelectChains = false
assumeStandardLibraryStripMargin = true

```

## Steps

Given code like this:
```scala
class TestSpec extends FlatSpec with Matchers {

"Foo" should "do things" in {
def bar: String = "test"
bar should endWith("st")
}

}
```

When I run scalafmt like this:

```bash
sbt test:scalafmt
```

or through IntelliJ IDEA.

## Problem

Scalafmt formats code like this:
```scala
class TestSpec extends FlatSpec with Matchers {

  "Foo" should "do things" in {
      def bar: String = "test"
      bar should endWith("st")
    }

}
```

## Expectation

I would like the formatted output to look like this:
```scala
class TestSpec extends FlatSpec with Matchers {

  "Foo" should "do things" in {
    def bar: String = "test"
    bar should endWith("st")
  }

}
```

**Comment by poslegm** [comment]
It works fine without `unindentTopLevelOperators = true` in config :thinking:

**Comment by tanishiking** [comment]
I think this has already fixed in v2.1.0-RC1 with https://github.com/scalameta/scalafmt/pull/1440 🙏
